### PR TITLE
[MI 2337] Fixed CI errors and review comments of PR #894

### PR DESCRIPTION
**Pull request by Nityanand13**
[PR #894](https://github.com/mattermost/mattermost-plugin-jira/pull/894)

### CODE REVIEW

**Review by Kshitij-Katiyar — APPROVED** [review]

File: server/client.go [hunk 1]
```diff
@@ -255,16 +255,16 @@ type AutoCompleteResult struct {
...
 
-type GroupItem struct {
+type JiraUserGroup struct {
 	Name string `json:"name"`
 }
 
-type GroupsInfo struct {
-	GroupsItem []GroupItem `json:"items"`
+type JiraUserGroupCollection struct {
+	JiraUserGroups []JiraUserGroup `json:"items"`
```
Comment (manojmalik20) [inline server/client.go:263]:
```suggestion
	JiraUserGroups []*JiraUserGroup `json:"items"`
```

File: server/client.go [hunk 2]
```diff
@@ -255,16 +255,16 @@ type AutoCompleteResult struct {
...
 }
 
 type CommentVisibilityResult struct {
-	Groups GroupsInfo `json:"groups"`
+	Groups JiraUserGroupCollection `json:"groups"`
```
Comment (manojmalik20) [inline server/client.go:267]:
```suggestion
	Groups *JiraUserGroupCollection `json:"groups"`
```

File: server/client.go [hunk 3]
```diff
@@ -286,7 +286,7 @@ func (client JiraClient) SearchCommentVisibilityFields(params map[string]string)
 	if err := client.RESTGet(commentVisibilityRoute, params, result); err != nil {
 		return nil, err
 	}
-	result.Groups.JiraUserGroups = append(result.Groups.JiraUserGroups, JiraUserGroup{visibleToAllUsers})
+	result.Groups.JiraUserGroups = append(result.Groups.JiraUserGroups, &JiraUserGroup{visibleToAllUsers})
```
Comment (manojmalik20) [inline server/client.go:289]: Have you tested the plugin after making these changes?

Comment (Nityanand13) [reply server/client.go:289]: Yeah, it is working perfectly fine.

**Pull request merged**